Casbin.UnitTests/ModelTests/EnforcerTest.cs

```diff
@@ -589,16 +589,16 @@ public void TestMultipleGroupTypeModelInMemory()
         IModel m = DefaultModel.Create();
         m.AddDef("r", "r", "sub, obj, act");
         m.AddDef("p", "p", "sub, obj, act");
-        m.AddDef("g", "g", "_, _");
+        m.AddDef("g", "g1", "_, _");
```

Comment:
Plz add new test for the issue case

Reply:
@Tanyuu reply to all review comments and resolve if needed

Reply:
@Tanyuu Maybe we can use the test case from the user at https://github.com/casbin/Casbin.NET/issues/329#issue-1851384512?